packages/@ember/-internals/glimmer/lib/components/textarea.ts

```diff
@@ -102,116 +49,91 @@ import layout from '../templates/empty';
-  * `key-press`
-
-  For example, if you desire an action to be sent when the input is blurred,
-  you only need to setup the action name to the event name property.
+  You could accomplish the same thing using classic invocation:
 
   ```handlebars
-  {{textarea focus-out="alertMessage"}}
+  {{textarea value=this.searchWord enter=this.query}}
   ```
 
-  See more about [Text Support Actions](/api/ember/release/classes/TextArea)
+  The main difference is that angle-bracket invocation supports any HTML attribute using HTML
+  attribute syntax, because attributes and arguments have different syntax when using angle-bracket
+  invocation. Curly brace invocation, on the other hand, only has a single syntax for arguments,
+  and components must manually map attributes onto component arguments.
 
-  ### Extension
+  When using classic invocation with `{{textarea}}`, only the following attributes are mapped onto
+  arguments:
 
-  Internally, `{{textarea}}` creates an instance of `TextArea`, passing
-  arguments from the helper to `TextArea`'s `create` method. You can
-  extend the capabilities of text areas in your application by reopening this
-  class. For example, if you are building a Bootstrap project where `data-*`
-  attributes are used, you can globally add support for a `data-*` attribute
-  on all `{{textarea}}`s' in your app by reopening `TextArea` or
-  `TextSupport` and adding it to the `attributeBindings` concatenated
-  property:
+  * rows
+  * cols
+  * name
+  * selectionEnd
+  * selectionStart
+  * autocomplete
+  * wrap
+  * lang
+  * dir
+  * value
 
-  ```javascript
-  import TextArea from '@ember/component/text-area';
+  ## Classic `layout` and `layoutName` properties
 
-  TextArea.reopen({
-    attributeBindings: ['data-error']
-  });
-  ```
+  Because HTML `textarea` elements do not contain inner HTML the `layout` and
+  `layoutName` properties will not be applied.
 
-  Keep in mind when writing `TextArea` subclasses that `TextArea`
-  itself extends `Component`. Expect isolated component semantics, not
-  legacy 1.x view semantics (like `controller` being present).
+  @method Textarea
+  @for Ember.Templates.components
+  @see {TextArea}
+  @public
+*/
 
-  See more about [Ember components](/api/ember/release/classes/Component)
+/**
+  See Ember.Templates.components.Textarea.
 
   @method textarea
   @for Ember.Templates.helpers
-  @param {Hash} options
+  @see {Ember.Templates.components.textarea}
   @public
 */
 
 /**
-  The internal class used to create textarea element when the `{{textarea}}`
-  helper is used.
-
-  See [Ember.Templates.helpers.textarea](/api/ember/release/classes/Ember.Templates.helpers/methods/textarea?anchor=textarea)  for usage details.
-
-  ## Layout and LayoutName properties
-
-  Because HTML `textarea` elements do not contain inner HTML the `layout` and
-  `layoutName` properties will not be applied.
+  The internal representation used for `Textarea` invocations.
 
   @class TextArea
```

Comment:
did this get associated to `@ember/component/text-area` correctly? does it need a `@for` or something